index.html

```diff
@@ -34,5 +34,206 @@ <h2 class="title">I'm a software developer</h2>
             <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
         </ul>
     </section>
+
+    <section class="works">
+    
+        <h2 class="title-one">My Recent Works</h2>
+        <hr class="line">
+        <section class="card1">
+            <img src="placeholder.png" alt="big picture">
+            <h3 class="post"> Multi-post stories</h3>
+            <p class="para1">A daily section of privately personalized<br>
+                reads: no account or sign-up are required.<br>
+                has been the industry standard's dummy<br>
+                text ever the 1500s, when an<br>
+                unknown printer took a standard dummy<br>
+                text. 
+            </p>
+            <div class="web">
+                <button class="css">css</button>
+                <button class="html">html</button>
+                <button class="boot">boostrap</button>
+                <button class="ruby">Ruby</button>
+            </div>
+            <button class="click">See Project</button>
+        </section>
+        <section class="card2">
+            <h3 class="data">Professional Art <br>
+                Printing Data
+            </h3>
+            <p class="para2">A daily section of privately<br>
+                personalized reads: no account or<br>
+                sign-up are required. has been<br>
+                the industry standard
+            </p>
+            <div class="web2">
+                <button class="html">html</button>
+                <button class="boot">boostrap</button>
+                <button class="ruby">Ruby</button>
+            </div>
+            <button class="click2">See Project</button>
+        </section>
+        <section class="card2">
+            <h3 class="data">Professional Art <br>
+                Printing Data
+            </h3>
+            <p class="para2">A daily section of privately<br>
+                personalized reads: no account or<br>
+                sign-up are required. has been<br>
+                the industry standard
+            </p>
+            <div class="web2">
+                <button class="html">html</button>
+                <button class="boot">boostrap</button>
+                <button class="ruby">Ruby</button>
+            </div>
+            <button class="click2">See Project</button>
```

Comment:
- [ ] Please note that the hover effect for the buttons suggested by the previous reviewer has not been implemented🙄, kindly see to it😊